### PR TITLE
Issue 2 makefile

### LINKED ISSUE #2: Create makefile to execute all notebooks and save figures

### LINKED ISSUE #7: Update README.md
- [x] Explain the requirements of some make commands.
E.g. si-covid.csv is required by parse_data.py, which is created from `make serial_interval` command.

- [x] Explain the requirement to have branchpro installed

### PR DESCRIPTION
**Pull request by FarmHJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,87 @@
 # transmission-heterogeneity-results
 
-This repository holds the results for the [branching process model](https://github.com/SABS-R3-Epidemiology/branchpro).
+This repository holds the results for the [branching process model](https://github.com/SABS-R3-Epidemiology/branchpro). This includes generating serial interval for COVID-19, parsing data from raw COVID-19 cases dataand reproducing figures from the paper.
```
Comment (rccreswell) [inline README.md]:
```suggestion
This repository holds the results for the [branching process model with imported cases](https://github.com/SABS-R3-Epidemiology/branchpro). This includes generating serial interval for COVID-19, parsing data from raw COVID-19 cases data and reproducing figures from the paper.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,87 @@
...
+4. `ontario_data` - to parse Ontario's data
+
+For example, to parse Australia's data:
+```bash
+make australia_data
+```
+
+
+## Figures
+To reproduce all figures:
```
Comment (rccreswell) [inline README.md:58]: Can you add a statement here saying that running the figures requires that the appropriate data and serial intervals have already been generated as described above

**Review by rccreswell — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (FarmHJ) [reply README.md:58]: Then I'll link #7 here. Solving both issues in a branch.

**Review by rccreswell — APPROVED** [review]
Thanks, this looks ready to me.
I just added a line to the readme about requiring jupyter.

**Pull request merged**